### ISSUE TITLE
Support for Container Queries

### ISSUE THREAD

**Issue by KoltonG** [frenic/csstype#175]
👋 Hey Team! Given that [**CSS Container Queries**](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Container_Queries) [is near supported in all major browsers](https://caniuse.com/css-container-queries), it would be great to have CSS typing for it.

I tried to fork and add it myself, however, the [mdn/data](https://github.com/mdn/data) repo does not include their typings. That repository also notes the following:

> Note We are in the process of deprecating the mdn/data package in favor of [w3c/webref](https://github.com/w3c/webref). If you depend on this project, let us know in our community [GitHub discussions](https://github.com/mdn/mdn-community/discussions/categories/platform). Thank you.

This might be the reason why it was not updated. The referred repository, [w3c/webref](https://github.com/w3c/webref), does have typing for CSS Container Queries via https://github.com/w3c/webref/blob/1a31f07e1814cf8919c0dd87023aaab20cd84a14/ed/css/css-contain-3.json

What would you recommend as the next steps to get CSS Container Queries typing into this package?

**Comment by meyer** [comment]
seems like the real issue here is "migrate mdn/data to "w3c/webref". the next step would probably be something along those lines.

**Comment by KoltonG** [comment]
Agreed @meyer! This does seem to be a much bigger rabbit hole 🐰 than simply adding the typing 🙈 Wondering if this was something the team was planning to do anyhow or is this the first request?

**Comment by bdow** [comment]
Looks like this may be fixed as of v 3.1.2. https://github.com/frenic/csstype/commit/40440cd290162aa2b21d9691b4d27e5e0fa038b3
Maybe can close now?

**Comment by valterkraemer** [comment]
Updating to 3.1.2 fixed it for me!
